### PR TITLE
Update to include grammar fixes and newer pot file

### PR DESCRIPTION
**Pull request by HelenCampbell**

### CODE REVIEW

File: locales/puppetlabs-mysql.pot [hunk 1]
```diff
@@ -23,152 +31,146 @@ msgstr ""
...
+"least 2)"
 msgstr ""
 
-#: ./lib/puppet/parser/functions/mysql_deepmerge.rb:30
+#: ../lib/puppet/parser/functions/mysql_deepmerge.rb:30
 msgid ""
-"mysql_deepmerge: unexpected argument type %{arg_class}, only expects hash argu"
-"ments"
+"mysql_deepmerge: unexpected argument type %{arg_class}, only expects hash "
+"arguments"
```
Comment (tphoney) [inline locales/puppetlabs-mysql.pot]: fullstop ?

File: locales/puppetlabs-mysql.pot [hunk 2]
```diff
@@ -23,152 +31,146 @@ msgstr ""
...
 msgstr ""
 
-#: ./lib/puppet/parser/functions/mysql_dirname.rb:9
+#: ../lib/puppet/parser/functions/mysql_dirname.rb:9
```
Comment (tphoney) [inline locales/puppetlabs-mysql.pot]: ../ is the wrong path 
./ is correct

Comment (tphoney) [reply locales/puppetlabs-mysql.pot]: same for all the ruby files

Comment (HelenCampbell) [reply locales/puppetlabs-mysql.pot]: Didn't notice it did that.. fixing now

**Comment by HelenCampbell** [comment]
@tphoney Updated with your changes

File: locales/puppetlabs-mysql.pot [hunk 3]
```diff
@@ -140,35 +139,38 @@ msgid "`name` `parameter` must match user@host/table format."
...
 
 #: ./lib/puppet/type/mysql_grant.rb:67
-msgid "`table` `property` for PROXY should be specified as proxy_user@proxy_host ."
+msgid ""
+"`table` `property` for PROXY should be specified as proxy_user@proxy_host."
 msgstr ""
 
-#: ./lib/puppet/type/mysql_grant.rb:96 ./lib/puppet/type/mysql_user.rb:29
-msgid "Invalid database user %{user}"
+#: ./lib/puppet/type/mysql_grant.rb:96 ../lib/puppet/type/mysql_user.rb:29
```
Comment (tphoney) [inline locales/puppetlabs-mysql.pot]: ../lib/puppet/type/mysql_user.rb:29
needs to be 
./lib/puppet/type/mysql_user.rb:29
There are more below

File: spec/unit/puppet/provider/mysql_database/mysql_spec.rb [hunk 4]
```diff
@@ -72,7 +72,7 @@
 
   describe 'exists?' do
     it 'checks if database exists' do
-      expect(instance.exists?).to be_truthy
+      expect(instance.exists?).to be_truthy # rubocop:disable RSpec/PredicateMatcher
```
Comment (tphoney) [inline spec/unit/puppet/provider/mysql_database/mysql_spec.rb]: can this not be fixed ?

Comment (yakirgb) [reply spec/unit/puppet/provider/mysql_database/mysql_spec.rb]: Maybe something like:
`expect(instance).to be_exists` ?

**Comment by pmcmaw** [comment]
LGTM 👍

**Pull request merged**